### ISSUE TITLE
iot controls level.color.temperature objects in group for SetPercentageRequests

### ISSUE THREAD

**Issue by Garfonso** [ioBroker/ioBroker.iot#71]
I have a lot of groups with two objects now. Those are all lamps which can be dimmed and can change color temperature. I'd like to control both with the same name using alexa. So I added the level.dimmer and level.color.temperature objects to the same group in the iot devices list.

It kind of works ok.
But my issue now is that every dim command also changes the color temperature accordingly. On the other hand changing the color temperature only changes the state of the color temperature object, as expected.

Why is this? I attach a log file. I added my voice commands to the file and removed some tokens and E-Mail stuff.
[alexaDimmenColorTemperature.txt](https://github.com/ioBroker/ioBroker.iot/files/3687940/alexaDimmenColorTemperature.txt)

Basically my question would be why iot controls the level.color.temperature object when receiving a SetPercentageRequest and how to avoid that. As I see it, level.color.temperature should only be controlled, if receving a "DecrementColorTemperatureRequest", "IncrementColorTemperatureRequest" and "SetColorTemperatureRequest". You can do all necessary manipulations with that requests. So reacting to SetPercentageRequests, too, should be deemed a bug.

**Comment by Leviathan09** [comment]
Hm just connected my three  Hue White Ambience Bulbs.
When i say "Alexa, schalte Flur auf 50%" everything is correct.


iot.0 | 2019-11-03 07:47:42.271 | debug | (5871) [ALEXA] Won't control zigbee.0.group_1.colortemp as we have a dimmer available for this channel.
-- | -- | -- | --
iot.0 | 2019-11-03 07:47:42.269 | debug | (5871) [ALEXA] Remember ON value for  "zigbee.0.group_1.brightness": 50
iot.0 | 2019-11-03 07:47:42.269 | debug | (5871) [ALEXA] Controlling zigbee.0.group_1.brightness .
iot.0 | 2019-11-03 07:47:42.267 | debug | (5871) [ALEXA] Won't control zigbee.0.group_1.state as we have a dimmer available for this channel.
iot.0 | 2019-11-03 07:47:42.263 | debug | (5871) [ALEXA] Controlling percentage: zigbee.0.group_1.colortemp
iot.0 | 2019-11-03 07:47:42.263 | debug | (5871) [ALEXA] Controlling percentage: zigbee.0.group_1.brightness
iot.0 | 2019-11-03 07:47:42.262 | debug | (5871) [ALEXA] Controlling percentage: zigbee.0.group_1.state
iot.0 | 2019-11-03 07:47:42.262 | debug | (5871) zigbee.0.group_1.brightness
iot.0 | 2019-11-03 07:47:42.262 | debug | (5871) [ALEXA] ALEXA Percent: Flur 50%

**Comment by Garfonso** [comment]
OK, I see. If the states are in the same channel, iot will stop processing the command.

But why try to control color temp with dimmer commands in the first place?

**Comment by Garfonso** [comment]
Still happening as before:
```
2020-08-07 10:00:45.161  - debug: iot.0 (5079) Request command/MAIL/alexa
2020-08-07 10:00:45.161  - debug: iot.0 (5079) Data: "{\"header\":{\"namespace\":\"Alexa.ConnectedHome.Control\",\"name\":\"SetPercentageRequest\",\"payloadVersion\":\"2\",\"messageId\":\"24f4f816-286d-43d2-a1e9-fd829245368e\"},\"payload\":{\"accessToken\":\"TOKEN\",\"appliance\":{\"applianceId\":\"Wohnzimmer_Test\",\"additionalApplianceDetails\":{\"group\":\"true\"}},\"percentageState\":{\"value\":50}}}"
2020-08-07 10:00:45.162  - debug: iot.0 (5079) 1596787245162 ALEXA: {"header":{"namespace":"Alexa.ConnectedHome.Control","name":"SetPercentageRequest","payloadVersion":"2","messageId":"24f4f816-286d-43d2-a1e9-fd829245368e"},"payload":{"accessToken":"TOKEN","appliance":{"applianceId":"Wohnzimmer_Test","additionalApplianceDetails":{"group":"true"}},"percentageState":{"value":50}}}
2020-08-07 10:00:45.162  - debug: iot.0 (5079) {"group":true,"channels":{"javascript.0.lamps.Wohnzimmer_TV":[{"id":"javascript.0.lamps.Wohnzimmer_TV.brightness","role":"level.dimmer","actions":["setPercentage","incrementPercentage","decrementPercentage","turnOn","turnOff"]}],"alias.0.lamps.Wohnzimmer_TV":[{"id":"alias.0.lamps.Wohnzimmer_TV.colortemp","role":"level.color.temperature","actions":["incrementColorTemperature","decrementColorTemperature","setColorTemperature"]}]},"smartTypes":{"alias.0.lamps.Wohnzimmer_TV.colortemp":"LIGHT"},"names":{"javascript.0.lamps.Wohnzimmer_TV.brightness":"lamps.Wohnzimmer_TV.brightness","alias.0.lamps.Wohnzimmer_TV.colortemp":"Wohnzimmer TV colortemp"},"byONs":{"javascript.0.lamps.Wohnzimmer_TV.brightness":"100","alias.0.lamps.Wohnzimmer_TV.colortemp":"stored"},"room":null,"func":null}
2020-08-07 10:00:45.162  - debug: iot.0 (5079) [ALEXA] New Request: SetPercentageRequest
2020-08-07 10:00:45.162  - debug: iot.0 (5079) [ALEXA] ALEXA Percent: Wohnzimmer_Test 50%
2020-08-07 10:00:45.162  - debug: iot.0 (5079) [ALEXA] DATA: [{"id":"javascript.0.lamps.Wohnzimmer_TV.brightness","role":"level.dimmer","actions":["setPercentage","incrementPercentage","decrementPercentage","turnOn","turnOff"]}]
2020-08-07 10:00:45.162  - debug: iot.0 (5079) javascript.0.lamps.Wohnzimmer_TV.brightness
2020-08-07 10:00:45.162  - debug: iot.0 (5079) [ALEXA] Controlling percentage: javascript.0.lamps.Wohnzimmer_TV.brightness
2020-08-07 10:00:45.163  - debug: iot.0 (5079) [ALEXA] DATA: [{"id":"alias.0.lamps.Wohnzimmer_TV.colortemp","role":"level.color.temperature","actions":["incrementColorTemperature","decrementColorTemperature","setColorTemperature"]}]
2020-08-07 10:00:45.163  - debug: iot.0 (5079) null
2020-08-07 10:00:45.163  - debug: iot.0 (5079) [ALEXA] Controlling percentage: alias.0.lamps.Wohnzimmer_TV.colortemp
2020-08-07 10:00:45.165  - debug: iot.0 (5079) [ALEXA] Controlling javascript.0.lamps.Wohnzimmer_TV.brightness .
2020-08-07 10:00:45.165  - debug: iot.0 (5079) [ALEXA] Remember ON value for  "javascript.0.lamps.Wohnzimmer_TV.brightness": 50
2020-08-07 10:00:45.166  - debug: iot.0 (5079) [ALEXA] Controlling alias.0.lamps.Wohnzimmer_TV.colortemp .
2020-08-07 10:00:45.166  - debug: iot.0 (5079) [ALEXA] Remember ON value for  "alias.0.lamps.Wohnzimmer_TV.colortemp": 4350

```

Probably this is intentional, though. Probably this is an edge case, too, where we can now direct users to use the alias-featuer and [devices adapter](https://github.com/ioBroker/ioBroker.devices)
